src/dynatrace/opentelemetry/metrics/export/__init__.py

```diff
@@ -37,6 +39,87 @@
 VERSION = "0.2.0b0"
 
 
+def _get_histogram_max(histogram: Histogram):
+    if math.isfinite(histogram.max):
```

Comment:
`isfinite` throws if `None` is used as the value. Not sure if it is possible to receive None as max but checking for None makes sense to me too

Reply:
In the Python SDK the way is to check for infinity as this is the default value if it is not present. However, it could be that this changes with the next Python Metrics SDK release, so I'm now checking for `None` now too ([4752e59](https://github.com/dynatrace-oss/opentelemetry-metric-python/pull/23/commits/4752e595b6fd6f45ebd289d914bd8b067a1594a5)). Thanks for catching this :slightly_smiling_face: